### ISSUE TITLE
Any chance of adding the remaining *arr packages?

### ISSUE THREAD

**Issue by aTosser** [xartin/gentoo-overlay#10]
I am particularly interested in prowlarr, as it replaces jackett with the ability to push indexer configurations to all *arr and some other applications. However currently readarr is missing as well as the *arr that should not be named. If not no biggie, I will just try to adapt your ebuilds to the packages and see if I can get them working.

**Comment by xartin** [comment]
so your interested in

prowlarr
readarr
bonarr

am I missing any or is the above summary correct?

prowlarr I've not looked at yet or at all but I could.
readarr i have the same situation as prowlarr regarding familiarity
bonarr is obsolete and hasn't been updated in many years. 

If Qstick hadn't recently updated lidarr I was becoming concerned it may never have been updated.

I have no quarrel adding ebuilds for upstream maintained packages. if the ebuilds are similar to radarr ( .net native linux core )  or sonarr ( still requires mono ) making new ebuilds for those and the group and user packages is a simple chore.

Until someone rises from the dead and updates bonarr it's a benefit for everyone if that's not easy to use with gentoo even from an overlay  considering this overlay is officially listed by sonarr for use with gentoo.

**Comment by xartin** [comment]
small update.

I have Prowlarr initial repo addition completed and ready to push. currently working on repo additions for readarr.

both of the above packages are dotnet core not relying on mono so the reusing the existing radarr ebuild for a template is a shoe in to attempt use with prowlarr and readarr.
 
bonarr will remain "boned" until someone updates five years of obsolete code.

**Comment by xartin** [comment]
initial prowlarr commit completed. testing still pending

initial readarr commit completed. testing still pending

I'll install these on a vm tomorrow sometime to test general functionality.

enjoy

**Comment by xartin** [comment]
Tested  installing and confirmed Prowlarr starts with both openrc and systemd.

Only one inconsequential warning was encountered with the openrc init script that could be corrected

`chown: cannot access '/etc/readarr': No such file or directory`

I suspect src_install() in the ebuilds is not creating package related /etc directories and that throws a warning on the first and possibly consecutive service starts.

Enjoy, that concludes the addition of prowlarr and readarr. I'll leave this bug open for any feedback of something is discovered.

**Comment by aTosser** [comment]
Yeah, readarr and prowlarr were my interests. Thanks much. I will report back if I encounter any issues.

**Comment by aTosser** [comment]
> Until someone rises from the dead and updates bonarr it's a benefit for everyone if that's not easy to use with gentoo even from an overlay considering this overlay is officially listed by sonarr for use with gentoo.

I do not really care but the wiki references a completely new package for the "other" content but there may be interest out there.

https://github.com/Whisparr/Whisparr/
